nsis/SectionCoreReqs.nsh

```diff
@@ -31,11 +31,11 @@
 	File                                            ..\bin\PCSX2_keys.ini.default
 
 	!insertmacro UNINSTALL.LOG_CLOSE_INSTALL
-
-    SetOutPath "$INSTDIR\Cheats"
+	
+	;/bin/Cheats needs CreateDirectory to prevent MakeNSIS from failing due to the folder being empty
     !insertmacro UNINSTALL.LOG_OPEN_INSTALL
-    File                                            ..\bin\Cheats\*
-    !insertmacro UNINSTALL.LOG_CLOSE_INSTALL
+    CreateDirectory                                 "$INSTDIR\Cheats"
```

Comment:
I'm not sure what you're talking about?

PCSX2 creates the cheats folder if it's missing, regardless of mode. The cheats folder in the install directory used to contain a Persona 4 pnach, but that was removed (#1239), so creating a cheats folder in the install directory now has no purpose.